Makefile

```diff
@@ -32,9 +29,7 @@ test:
 clean:
 	rm -f $(MICROSERVICES)
 
-docker: $(DOCKERS)
-
-docker_device_rfid_llrp_go:
+docker:
 	docker build \
 		--build-arg http_proxy \
```

Comment:
Still produces image named `docker-device-rfid-llrp-go` rather than `device-rfid-llrp`

Reply:
yep, I found what I did wrong. made the replacement and tested it. I think it should be fixed now